metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -115,6 +115,23 @@ case class ScalafixProvider(
       }
     }
   }
+  private def scalafixSucceded(evaluation: ScalafixEvaluation): Boolean =
+    evaluation.isSuccessful && evaluation
+      .getFileEvaluations()
+      .headOption
```

Comment:
I think you can just skip this `.headOption` since whether or not it's empty won't effect the `.forAll(_.isSuccessful)` below it.